intake/source/discovery.py

```diff
@@ -84,21 +83,34 @@ def scanned(self):
             self._scanned = _package_scan()
         return self._scanned
 
-    def enabled_plugins(self):
+    @property
```

Comment:
Not that I mind, but was there a particular reason for turning these into properties? It makes the diff that bit longer.

Reply:
Oh good question. I assumed you wouldn't be able to perform direct operations on plain methods (`self.disabled().append(plugin)` for example) but I didn't test that assumption!

Reply:
Yes, you totally can, since it gives you a mutable container (list?). With properties, it would be more appropriate to have a setter method, I suppose, but far too verbose here.

Reply:
Nice, TIL! I'll turn them back into methods :)